test/rng_testsuite/conformance_tests/minstd_rand_rand0_test.pass.cpp

```diff
@@ -27,6 +27,15 @@
 //     The 10000th consecutive invocation of a default-constructed object of type minstd_rand
 //     produces the value 399268537
 
+#if (!defined(_ONEDPL_BACKEND_SYCL) || (_ONEDPL_BACKEND_SYCL == 0))
```

Comment:
Since `_ONEDPL_BACKEND_SYCL` is an internal macro I'd suggest to use public name: `ONEDPL_USE_DPCPP_BACKEND` :
```
#if (ONEDPL_USE_DPCPP_BACKEND == 0)
```
If the macro is not defined it means that it equals 1.
BTW, is it expected that the test will run with non-SYCL compilers?

Reply:
These RNG tests should be skipped in case of non-SYCL compilers. 
Andrey, I can see that other tests also use _ONEDPL_BACKEND_SYCL macro. E.g.: https://github.com/oneapi-src/oneDPL/blob/main/test/general/type_requirements.pass.cpp
Should we be aligned here?

Reply:
Yes, honestly saying other tests should also use public macros. I'd suggest to create a special macro in test/support/pstl_test_config.h (file also should be renamed to `test_config.h`), e.g.
```
#define SKIP_NON_SYCL_TEST (ONEDPL_USE_DPCPP_BACKEND == 0)
```
to not be dependent on name changes for public macro if any. Then we can use such macro for other tests.
BTW, if we're going to skip testing for non-SYCL compilers, I think we should rely on Compiler's macros, e.g.
```
#define SKIP_NON_SYCL_TEST (defined(CL_SYCL_LANGUAGE_VERSION) || defined(SYCL_LANGUAGE_VERSION))
```

Reply:
I can rename pstl_test_config.h file and create such define - but there will be changes in 94 files (in PSTL tests mostly). 

Should we do these changes in the separate pull request? Or we can proceed with the current one?

Reply:
I'd suggest to start with the macro here and use for your tests. Renaming of the file and replacing internal macros in other tests can be done separately

Reply:
I have changed define to the public one + refactored a little bit RNG test skipping